packages/article/package.json

```diff
@@ -15,7 +15,7 @@
   },
   "dependencies": {
     "@thetimes/with-client": "1.0.7",
-    "@times-components/article": "0.62.2",
+    "@times-components/article": "0.62.13",
     "@times-components/provider": "0.26.0",
```

Comment:
Might be safer to bump it all: including provider and author-profile

Reply:
Ah actually I need the new provider as well, cause we need to add the fields to the query, well spotted.

Reply:
For the author profile, that shouldn't be an issue, let's try without first.

Reply:
Linking tends to get messed up with multiple different versions but let's give it a try.
The problem is, almost every package in times-components uses markup, and markup uses almost all packages, so they all have dependencies on each other.